### PR TITLE
26 debug frmelementdesigner paints component out of view

### PR DESCRIPTION
**Pull request by WilfridoCO**
I've made a PointF variable called "puntoCentro", it's the center of PicDraw, I use in the Paint parameters.
I've undone the changes that I did in Component.cs, now it's exactly the same as the original one.

### CODE REVIEW

File: SimpleAnnPlayground/Debugging/FrmElementDesigner.Designer.cs [hunk 1]
```diff
@@ -104,8 +104,8 @@ private void InitializeComponent()
             // 
             // PicDraw
             // 
+            this.PicDraw.Anchor = System.Windows.Forms.AnchorStyles.None;
             this.PicDraw.BackColor = System.Drawing.Color.White;
-            this.PicDraw.Dock = System.Windows.Forms.DockStyle.Fill;
```
Comment (omarisai) [inline SimpleAnnPlayground/Debugging/FrmElementDesigner.Designer.cs:108]: The `DockStyle` property for `PicDraw` should remain as `System.Windows.Forms.DockStyle.Fill`.

To get an updated value of the `PicDraw` center is necessary to handle [PicDraw_Resize()](https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.control.resize?view=windowsdesktop-6.0) event.

File: SimpleAnnPlayground/Debugging/FrmElementDesigner.Designer.cs [hunk 2]
```diff
@@ -104,8 +104,8 @@ private void InitializeComponent()
             // 
             // PicDraw
             // 
+            this.PicDraw.Anchor = System.Windows.Forms.AnchorStyles.None;
```
Comment (omarisai) [inline SimpleAnnPlayground/Debugging/FrmElementDesigner.Designer.cs:107]: This should be removed to use the `Dock` property.

File: SimpleAnnPlayground/Graphical/Models/Component.cs [hunk 3]
```diff
@@ -286,4 +286,4 @@ internal void Paint(Graphics graphics, PointF location, State state = State.None
             graphics.TranslateTransform(Center.X - location.X, Center.Y - location.Y);
         }
     }
-}
+}
```
Comment (omarisai) [inline SimpleAnnPlayground/Graphical/Models/Component.cs:289]: This shouldn't be committed.

File: SimpleAnnPlayground/Debugging/FrmElementDesigner.cs [hunk 4]
```diff
@@ -222,6 +225,7 @@ private void DeserializeData(string content)
 
             LstModes.SelectedIndex = 0;
             PgdProperties.SelectedObject = _component;
+
```
Comment (omarisai) [inline SimpleAnnPlayground/Debugging/FrmElementDesigner.cs:228]: This shouldn't be committed.

File: SimpleAnnPlayground/Debugging/FrmElementDesigner.cs [hunk 5]
```diff
@@ -165,12 +166,14 @@ private void PicDraw_Paint(object sender, PaintEventArgs e)
                 }
             }
 
-            _component.Paint(e.Graphics, PointF.Empty, _state, LstConnectors.SelectedItem as Connector);
+            PointF puntoCentro = new PointF(PicDraw.Height / 2, PicDraw.Width / 2);
```
Comment (omarisai) [inline SimpleAnnPlayground/Debugging/FrmElementDesigner.cs:169]: Please change the variable name to a name in the English language.

File: SimpleAnnPlayground/Debugging/FrmElementDesigner.Designer.cs (hunk 1 above)
Comment (WilfridoCO) [reply SimpleAnnPlayground/Debugging/FrmElementDesigner.Designer.cs:108]: Oh, you're right, when I started I moved some options in the properties box, I didn't realise that I didn't get those back, not problem, I'll fix them. Thanks!

File: SimpleAnnPlayground/Debugging/FrmElementDesigner.Designer.cs (hunk 2 above)
Comment (WilfridoCO) [reply SimpleAnnPlayground/Debugging/FrmElementDesigner.Designer.cs:107]: Same case, I'll fix it

**Review by omarisai — APPROVED** [review]

**Pull request merged**